src/exojax/spec/atomll.py

```diff
@@ -0,0 +1,408 @@
+    for i, mask in enumerate(QTmask):
+        QTref[i] = QTref_284[mask]
+        
+    #Use Irwin_1981 for Fe I (mask==76)  #test211013Tako
+    if Irwin==True:
+        QTref[np.where(QTmask == 76)[0]] = atomllapi.partfn_Fe(Tref)
+
+    S0 = -A*gupper*np.exp(-hcperk*elower/Tref)*np.expm1(-hcperk*nu_lines/Tref)\
+        /(8.0*np.pi*ccgs*nu_lines**2*QTref)
+
+    return(S0)
+
+
+
+def gamma_vald3(T, PH, PHH, PHe, ielem, iion, \
+    nu_lines, elower, eupper, atomicmass, ionE, \
+    gamRad, gamSta, vdWdamp, enh_damp=1.0): #, vdW_meth="V"):
+    """HWHM of Lorentzian (cm-1) caluculated as gamma/(4*pi*c) [cm-1] for lines with the van der Waals gamma in the line list (VALD or Kurucz), otherwise estimated according to the Unsoeld (1955)
+
+    Args(inputs):
+      T: temperature (K)
+      PH: hydrogen pressure (bar) #1 bar = 1e6 dyn/cm2
+      PHH: H2 molecule pressure (bar)
+      PHe: helium pressure (bar)
+      ielem:  atomic number (e.g., Fe=26)
+      iion:  ionized level (e.g., neutral=1, singly ionized=2, etc.)
+      nu_lines:  transition waveNUMBER in [cm-1] (NOT frequency in [s-1])
+      elower: excitation potential (lower level) [cm-1]
+      eupper: excitation potential (upper level) [cm-1]
+      atomicmass: atomic mass [amu]
+      ionE: ionization potential [eV]
+      gamRad: log of gamma of radiation damping (s-1) #(https://www.astro.uu.se/valdwiki/Vald3Format)
+      gamSta: log of gamma of Stark damping (s-1)
+      vdWdamp:  log of (van der Waals damping constant / neutral hydrogen number) (s-1)
+      enh_damp: empirical "enhancement factor" for classical Unsoeld's damping constant
+          #cf.) This coefficient (enh_damp) depends on  each species in some codes such as Turbospectrum. #tako210917
+
+    Args(calculated):
+      chi_lam (=h*nu=1.2398e4/wvl[AA]): energy of a photon in the line
+      C6: interaction constant (Eq.11.17 in Gray2005)
+      logg6: log(gamma6) (Eq.11.29 in Gray2005)
+      gam6H: 17*v**(0.6)*C6**(0.4)*N
+           #(v:relative velocity, N:number density of neutral perturber)
+      Texp: temperature dependency (gamma6 \sim T**((1-α)/2) ranging 0.3–0.4)
+
+    Returns:
+      gamma: pressure gamma factor (cm-1)
+
+    Notes:
+    "/(4*np.pi*ccgs)" means:  damping constant -> HWHM of Lorentzian in [cm^-1]
+    
+    Reference of van der Waals damping constant (pressure/collision gamma):
+      Unsöld1955: https://ui.adsabs.harvard.edu/abs/1955psmb.book.....U
+      Kurucz+1981: https://ui.adsabs.harvard.edu/abs/1981SAOSR.391.....K
+      Barklem+1998: https://ui.adsabs.harvard.edu/abs/1998MNRAS.300..863B
+      Barklem+2000: https://ui.adsabs.harvard.edu/abs/2000A&AS..142..467B
+      Gray+2005: https://ui.adsabs.harvard.edu/abs/2005oasp.book.....G
+    """
+    ccgs = 2.99792458e10 #[cm/s]
+    kcgs = 1.38064852e-16 #[erg/K]
+    gamRad = np.where(gamRad==0., -99, gamRad)
+    gamSta = np.where(gamSta==0., -99, gamSta)
+    chi_lam = nu_lines/8065.54 #[cm-1] -> [eV]
```

Comment:
I found many 8065.54 in this file. Can you define it in  [utils.constants](https://github.com/HajimeKawahara/exojax/blob/develop/src/exojax/utils/constants.py) and use it?